lib/rock/webapp/tasks/api.rb

```diff
@@ -102,6 +102,55 @@ def get_operation(name_service, name, operation_name)
                         Hash[task: task_by_name(params[:name_service], params[:name]).to_h]
                     end
                     
+                    desc "starts a task"
+                    get ':name_service/:name/start' do
+                        task = task_by_name(params[:name_service], params[:name])
+                        begin
+                            task.start
+                            return task.rtt_state == :RUNNING
+                        rescue Orocos::StateTransitionFailed => exception
+                            puts exception.pretty_inspect
```

Comment:
Would be nicer to send the exception through the error. Bottom line: no puts on the server side.